warehouse/forklift/legacy.py

```diff
@@ -1384,6 +1410,15 @@ def file_upload(request):
             ):
                 release_url.verified = True
 
+        if release.home_page and not release.home_page_verified and home_page_verified:
+            release.home_page_verified = True
+        if (
+            release.download_url
+            and not release.download_url_verified
+            and download_url_verified
+        ):
+            release.download_url_verified = True
+
```

Comment:
I don't think there's a case where `release.home_page` would be falsey but `home_page_verified` would be true, so this can just be:

```suggestion
        if home_page_verified and not release.home_page_verified:
            release.home_page_verified = True
        if download_url_verified and not release.download_url_verified:
            release.download_url_verified = True

```